decidim-assemblies/app/commands/decidim/assemblies/admin/notify_role_assigned_assembly.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Assemblies
+    module Admin
+      # A command to notify users when a role is assigned for a participatory process
+      class NotifyRoleAssignedAssembly < Rectify::Command
```

Comment:
Let's add the preposition, this will affect the class and the file names:
```suggestion
      class NotifyRoleAssignedToAssembly < Rectify::Command
```

also the documentation for this class uses "a participatory process" when it should say "an assembly", correct?

Reply:
nice catch, did not realise, fixed it :)